resources-compose/src/jvmMain/kotlin/dev/icerock/moko/resources/compose/toImageBitmap.kt

```diff
@@ -1,23 +0,0 @@
-/*
- * Copyright 2021 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
- */
-
-package dev.icerock.moko.resources.compose
-
-import androidx.compose.ui.graphics.ImageBitmap
-import androidx.compose.ui.graphics.asImageBitmap
-import org.jetbrains.skija.Bitmap
-import org.jetbrains.skija.ColorAlphaType
-import org.jetbrains.skija.ColorType
-import org.jetbrains.skija.ImageInfo
-import java.awt.image.BufferedImage
-import java.awt.image.DataBufferByte
-
-// TODO this needs to be updated, would be nice to have a converter in compose
-fun BufferedImage.toImageBitmap(): ImageBitmap {
```

Comment:
why this removed? it can be used at this moment somewhere in projects. backward compatibility broken by this change - need to revert

Reply:
Done